plugins/workspace-plugin/tests/file-uri.spec.ts

```diff
@@ -8,6 +8,8 @@
  * SPDX-License-Identifier: EPL-2.0
  ***********************************************************************/
 
+import 'jest';
```

Comment:
looks odd to import `jest` ?

Reply:
This was the only way I could get rid of the errors reported by the typescript plugin.

Reply:
but jest is added on tsconfig types

Reply:
no idea ... 
![Selection_142](https://user-images.githubusercontent.com/650571/104023785-70a07000-51c2-11eb-840d-c5b046fdac11.png) maybe a bug in typescript plugin.

Do you want to remove these imports?

Reply:
could you add
```
      "types": ["reflect-metadata", "jest"],
```

in https://github.com/eclipse/che-theia/blob/master/plugins/workspace-plugin/tsconfig.json ?

like it's done there: https://github.com/eclipse/che-theia/blob/master/plugins/ports-plugin/tsconfig.json#L12-L14

Reply:
I tried that already (see the M in tsconfig.ts in the screenshoft above). And just retried but it doesn't get rid of the error message. Maybe a bug in the plugin.

Reply:
@benoitf I rebased and removed the imports